cggmp21/src/signing.rs

```diff
@@ -425,6 +483,20 @@ where
     };
     debug_assert_eq!(key_share.core.shared_public_key, X.iter().sum::<Point<E>>());
 
+    // Apply additive shift
+    let shift = additive_shift.unwrap_or(Scalar::zero());
+
+    let Shift = Point::generator() * shift;
```

Comment:
Super-nitpick: maybe remove this empty line?

Reply:
```suggestion
    let shift = additive_shift.unwrap_or(Scalar::zero());
    let Shift = Point::generator() * shift;
```